app/entities/user.go

```diff
@@ -46,6 +47,9 @@ func (u User) Validate() error {
 		return err
 	}
 
+	h := sha256.Sum256([]byte(u.Password))
```

Comment:
Why do we keep it inside Validate func?

Reply:
Please tell why not. The concept was to keep naked password for less time, and solve the hashing with less changes in the project. It was done.

Reply:
Because it doesn't follow the `Validate()` function intention. Hashing of a password - is not a validation process.

Reply:
What about renaming it to ValidateAndHash?

Reply:
Please read about single responsibility principle.

Reply:
Now the hash function has been renamed and has a single responsibility